nixtract/model.py

```diff
@@ -27,9 +42,9 @@ class NixpkgsMetadata(BaseModel):
         default=None,
         description="Flag indicating whether the derivation is broken",
     )
-    license: str | None = Field(
+    license: list[License] | None = Field(
```

Comment:
By convention I use plurals for lists
```suggestion
    licenses: list[License] | None = Field(
```